Yoast/wordpress-seo
# changed placeholder text

## Summary

This PR can be summarized in the following changelog entry:

* Changed meta description placeholder logic for the snippet preview. If no specific meta description has been typed, a placeholder message will always be shown.

## Relevant technical choices:

* Undid many additions made by https://github.com/Yoast/wordpress-seo/issues/9752

## Test instructions

This PR can be tested by following these steps:

* Use with sibling pr: https://github.com/Yoast/yoast-components/pull/580
* Check if the meta desc in the snippet preview is the placeholder if none is typed. And if it is the typed description if one was typed.
* Check if the snippet variables are still properly replaced (code was touched in this area).
* Check if the unit-tests pass

## Quality assurance


Fixes #9930
